Automattic/wp-calypso
# G Suite Basic Component Refactor

#### Changes proposed in this Pull Request

* G Suite Basic Components refactored into `GSuiteFeatures` & `GSuiteLearnMore`

#### Testing instructions

1. Enter "Add Domain Flow" on existing sites with branch
1. Get to "Add G Suite" step.
1. Verify that branch version & WordPress.com version are almost the same functionally
1. Navigate to email management  for a site and domain without G Suite
1. Verify that branch version & WordPress.com version are almost the same functionally
1. Navigate to G Suite Cancel Dialog Features step for Existing Purchase
1. Verify that branch version & WordPress.com version are almost the same functionally
